dockerfiles/jupyter/Dockerfile

```diff
@@ -57,7 +57,8 @@ RUN cd /tmp/mlrun && python -m pip install ".[complete-api]"
 
 # This will usually cause a cache miss - so keep it in the last layers
 ARG MLRUN_CACHE_DATE=initial
-RUN git clone --branch 1.3.x https://github.com/mlrun/demos.git $HOME/demos && \
+RUN git clone --branch 1.5.x https://github.com/mlrun/demos.git $HOME/demos && \
```

Comment:
why 1.5.x and not development?

Reply:
development branch in demos isn't aligned 
@liranbg accepted 1.5.x